hw03_samundra.py

```diff
@@ -0,0 +1,110 @@
+#name:Samundra Adhikari
+#username: adhikaris2
+#googledocs: https://docs.google.com/document/d/1Mj5nshMpoanpbSKld58QdK__H9CcjM_WI2m9WeCyrzo/edit#heading=h.66gmzdtk59q2
+# Drawing a cat.
+#Acknowlege: Used ChatGPT to ask questions, and used textbook.
+
+import turtle
+
+# Draw the cat's head
+
+def draw_head(cat, x,y):
```

Comment:
Why do you have input parameters `x` and `y`? They aren't used in this function or most of the ones below, either.

Reply:
I have fixed these issues.